modules/compute/gke-node-pool/variables.tf

```diff
@@ -96,7 +96,7 @@ variable "local_ssd_count_ephemeral_storage" {
   [See above](#local-ssd-storage) for more info.
   EOT 
   type        = number
-  default     = 0
+  default     = null
```

Comment:
null as an int can be a bit of an anti-pattern. I want to understand the driver for this. What behavior drove us to make this change.

Reply:
description updated.